css/styles.css

```diff
@@ -188,3 +201,199 @@ Headline
+  border-radius: var(--border-radius);
+  overflow: hidden;
+}
+
+.card__image-container {
+  position: relative;
+}
+
+.card__image-container::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  aspect-ratio: 1;
+  content: "";
+  background-color: #ebf0eea7;
+  transition: background-color 400ms ease-in-out;
+}
+
+.works__card:hover > .card__image-container::before {
+  background-color: #ebf0ee20;
+}
+
+.card__image {
+  object-fit: cover;
+  height: 250px;
+}
+
+.card__content {
+  padding: 1rem;
+  border-radius: var(--border-radius);
+  background-color: #fff;
+  translate: 0 -10px;
+}
+
+.card__title {
+  color: #3a4a42;
+  font-size: 2rem;
+  text-align: center;
+}
+
+.card__technologies {
+  color: #3a4a42;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  padding-left: 0;
+  list-style-type: none;
+  margin-block: 1rem;
+}
+
+.card__technologies li {
+  border-radius: var(--border-radius);
+  padding: 0.4rem 0.6rem;
+  font-weight: 500;
+  background-color: #ebf0ee;
+  color: #000;
+  width: max-content;
+}
+
+.card__button,
+.aboutme__resume {
+  font-size: 1.3rem;
+  background-color: #36b37f;
+  color: #fff;
+  border: none;
+  outline: none;
+  border-radius: var(--border-radius);
+  padding: 0.8rem 1rem;
+  display: block;
+  margin-inline: auto;
+  transition: box-shadow 250ms ease-in-out;
+}
+
+/*
+=====================================
+About me section
+=====================================
+*/
+
+.aboutme__header {
+  padding: 0 var(--horizontal-spacing);
+  min-height: 70vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url("../img/decorators/about_me/origami-bird.png"), url("../img/decorators/about_me/origami-ship.png");
+  background-position: 115% 8%, -25% 84%;
+  background-size: 120px, 140px;
+  background-repeat: no-repeat;
+}
+
+.aboutme__description {
+  text-align: center;
+  line-height: 2rem;
+}
+
+.aboutme__resume {
+  text-decoration: none;
+  border-radius: var(--border-radius);
+  width: max-content;
+  margin-block: 1rem;
+}
+
+.card__button:hover,
+.aboutme__resume:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.card__button:active,
+.aboutme__resume:active {
+  background-color: #008552;
+}
+
+.aboutme__cards {
+  padding: 0 var(--horizontal-spacing);
+  display: grid;
+  gap: 1.5rem;
+  margin-bottom: 5rem;
+}
+
+.aboutme__card {
+  background-color: #ebf0ee;
+  padding: 2rem;
+  border-radius: var(--border-radius);
+}
+
+.card__icon {
+  margin: 1rem 0 5rem 0;
+  background-color: #fff;
+  border-radius: 50%;
+  padding: 1rem;
+  width: max-content;
+  margin-inline: auto;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+}
+
+.card__topic {
+  font-size: 1.7rem;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+  margin-bottom: 1.5rem;
+}
+
+.card__list {
+  padding-left: 0;
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+}
```

Comment:
- [ ] Kindly make sure the LANGUAGES, SKILLS, AND FRAMEWORKS list is aligned in the center of the card container in order to meet the provided Figma design.

    - ### YOUR DESIGN
      ![image](https://user-images.githubusercontent.com/87197412/205188860-52139486-643b-4f77-a3b5-41acbd857654.png)

    - ### BETTER DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/205188889-67fd4b60-615f-464a-986c-10c2119a07b5.png)